### PR TITLE
Feature/api calls

### LINKED ISSUE #2: Display photo of the day info
**Describe the solution you'd like**
As a user, I should be able to see the photo of the day information, so that I can enjoy the information.

**Additional context**
Should be able to see:
[ ] Photo of the Day
[ ] Photo title
[ ] Photo credit
[ ] Photo Explanation

### PR DESCRIPTION
**Pull request by richardltyler**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- uses data fetched from the api instead of dummy data 

## *Instructions for reviewer:*
*Where should the reviewer start?*
- app.vue

*How should this be manually tested?*
- load the app and see that the displayed photo has today's date on it 

*Any background context you want to provide?*
*What are the relevant tickets?*
closes #2 #3

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -20,13 +20,22 @@
...
     PicOfTheDay
+  },
+  data() {
+    return {
+      potd: {}
+    }
+  },
+  created() {
+    apiCalls.getTodaysPic()
+      .then(photo => this.potd = photo);
```
Comment (colefiscus) [inline src/App.vue:38]: Excellent work!

File: src/PicOfTheDay.vue [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
       webkitallowfullscreen="webkitallowfullscreen"
     />
-    <img :src="url" class='photo' v-if="media_type === 'image'" :alt="title">
-    <h2>{{ date }}</h2>
-    <h3>{{ title }}</h3>
-    <h4>{{ explanation }}</h4>
+    <img :src="potd.url" class='photo' v-if="potd.media_type === 'image'" :alt="potd.title">
+    <h2>{{ potd.date }}</h2>
+    <h3>{{ potd.title }}</h3>
+    <h4>{{ potd.explanation }}</h4>
```
Comment (colefiscus) [inline src/PicOfTheDay.vue]: Is this how these work? I thought the v-bind: was used on data properties, like :src would be a data property. Here we only have potd as a property so is that binding at all inside of the template?

File: src/PicOfTheDay.vue [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
-      explanation: "What would it look like to land on Mars?  To better monitor the instruments involved in the Entry, Decent, and Landing of the Perseverance Rover on Mars last week, cameras with video capability were included that have now returned their images. The featured 3.5-minute composite video begins with the opening of a huge parachute that dramatically slows the speeding spacecraft as it enters the Martian atmosphere.  Next the heat shield is seen separating and falls ahead. As Perseverance descends, Mars looms large and its surface becomes increasingly detailed.  At just past 2-minutes into the video, the parachute is released and Perseverance begins to land with dust-scattering rockets. Soon the Sky Crane takes over and puts Perseverance down softly, then quickly jetting away. The robotic Perseverance rover will now begin exploring ancient Jezero Crater, including a search for signs that life once existed on Earth's neighboring planet.",
-      media_type: "video",
-      service_version: "v1",
-      title: "Video: Perseverance Landing on Mars",
-      url: "https://www.youtube.com/embed/4czjS9h4Fpg?rel=0"
-    }
-  }
+  name: 'PicOfTheDay',
+  props: ['potd'],
```
Comment (colefiscus) [inline src/PicOfTheDay.vue]: Are props always in an array?

File: src/apiCalls.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+  getSpecificDatesPhoto(date) {
+    return (
+      fetch(`https://api.nasa.gov/planetary/apod?date=${date}&api_key=j9VLjGbdXCRXtf61nCle9dLGtNzWVnNqUM1BNV86`)
+        .then(response => response.json())
+    )
+  }
+}
+
+export default apiCalls;
```
Comment (colefiscus) [inline src/apiCalls.js:17]: Looks good!

**Review by lbmerchant93 — APPROVED** [review]
Will the fetch call need to be moved to the POD component? If we are considering the POD component to be a route (home for right now) then we wouldn't be calling the component in the App.vue. Is there a way to pass the props into the route in Vue like in React?

**Review by colefiscus — APPROVED** [review]
Looks great!!

File: src/PicOfTheDay.vue (hunk 2 above)
Comment (richardltyler) [reply src/PicOfTheDay.vue]: If you're assigning dynamic values to common html attributes, you have to bind them by using `v-bind:` or `:` for short

File: src/PicOfTheDay.vue (hunk 3 above)
Comment (richardltyler) [reply src/PicOfTheDay.vue]: Yep! And they should always be put between quotes

**Pull request merged**